lib/graphiti/sideload.rb

```diff
@@ -224,6 +224,11 @@ def parent_resource
       @parent_resource ||= parent_resource_class.new
     end
 
+    def clear_resources
```

Comment:
Could we add a link to the original issue for this, just to jog memory when we come across this code later on?

Reply:
Added in 46453e71f33592b50936c88e3db65645d357f1bd